### DISCUSSION TITLE
Nearly a week, Still under review status on externship dashboard

### DISCUSSION THREAD

**Discussion by madrigal1** [github/india#632] [Students]
Nearly a week, yet no change in the status or follow-up emails. Has anyone been accepted, received any follow-up communication? Also does anyone know any contact info to enquire further

**Comment by Harsh151200** [comment]
Same here

**Comment by punitjain03** [comment]
Hi @madrigal1 , Currently we are reviewing the profiles, From today EOD or tomorrow morning, We will notify the status of the application status. Post that you can apply for the organisations.

**Reply by madrigal1** [reply]
> Hi @madrigal1 , Currently we are reviewing the profiles, From today EOD or tomorrow morning, We will notify the status of the application status. Post that you can apply for the organisations.

Thanks for the reply @punitjain03

**Reply by parthsarthi313** [reply]
> Hi @madrigal1 , Currently we are reviewing the profiles, From today EOD or tomorrow morning, We will notify the status of the application status. Post that you can apply for the organisations.

Hey still it shows under review

**Reply by santanukumar666** [reply]
My profile is still under review

**Reply by Rajatino16** [reply]
My profile is also under review, even though it's been more than a week since I had applied.

**Reply by hariketsheth** [reply]
Yes, Even my profile is still under review, I had applied on the opening date itself. 

Kindly please look into it. 
Since we need to submit proposals also before 14th December

**Reply by YashwanthAlluri** [reply]
even my profile has been under review for more than a week can anyone check it

**Comment by YashwanthAlluri** [comment]
My application has been under review for more than a week now. Can anyone check it out?

**Comment by punitjain03** [comment]
Stop spamming please
